src/dataset-builder/DatasetBuilder.ts

```diff
@@ -382,31 +372,11 @@ export const ConceptSetSelector = ({
   onStateChange: OnStateChangeHandler;
 }) => {
   return h(Selector<ConceptSet>, {
-    headerAction: h(
-      Link,
-      {
-        onClick: () => onStateChange(conceptSetCreatorState.new(_.map(toConcept, conceptSets))),
-        'aria-label': 'Create new concept set',
-      },
-      [icon('plus-circle-filled', { size: 24 })]
-    ),
     number: 2,
     onChange,
     objectSets: [
-      {
-        header: 'Concept sets',
-        values: conceptSets,
-        makeIcon: (value, header) =>
-          h(
-            Clickable,
-            {
-              'aria-label': `Delete ${header}/${value.name}`,
-              onClick: () => updateConceptSets(_.without([value])),
-            },
-            [icon('trash-circle-filled', { size: 20 })]
-          ),
-      },
-      { header: 'Prepackaged concept sets', values: prepackagedConceptSets ?? [] },
+      { header: '', values: conceptSets },
+      { header: '', values: prepackagedConceptSets ?? [] },
```

Comment:
is there a better solution than passing in empty strings?

Reply:
I would say a better option is to make `header` nullable on the `HeaderAndValues` type, and then do the undefined checking at the usages (`SelectorComponent` and `ObjectSetListSection`) - I think that would be good to do anyway cause right now we're finding the group by the header, and duplicating headers like this has a lot of risk of making it so that it doesn't actually find the group. See https://github.com/DataBiosphere/terra-ui/pull/4902/files#diff-7b85652f508bf795d1b7dbe94b817d2e702e2b5182074c8f7712ec3c2f10edc0R197